src/components/signup-view/signup-view.jsx

```diff
@@ -0,0 +1,72 @@
+import { useState } from "react";
+
+export const SignupView = () => {
+  const [username, setUsername] = useState("");
+  const [password, setPassword] = useState("");
+  const [email, setEmail] = useState("");
+  const [birthday, setBirthday] = useState("");
```

Comment:
If you are keeping it as a string this is fine, but if you want it as a datatime start with useState(new Date())